tools/go.mod

```diff
@@ -0,0 +1,189 @@
+module github.com/ubuntu/oidc-broker/tools
+
+go 1.21.5
```

Comment:
shouldn't this be ?

```
go 1.21.0
toolchain go1.21.5 
```

Reply:
I followed what we [have in authd](https://github.com/ubuntu/authd/blob/main/tools/go.mod), but I can update to this new format.